src/main/java/io/percy/appium/providers/AppAutomate.java

```diff
@@ -4,21 +4,20 @@
 
 import io.appium.java_client.AppiumDriver;
 import io.percy.appium.AppPercy;
-import io.percy.appium.lib.Cache;
 
 public class AppAutomate extends GenericProvider {
     private AppiumDriver driver;
     private Boolean markedPercySession = true;
-    private static String sessionId;
 
     public AppAutomate(AppiumDriver driver) {
         super(driver);
         this.driver = driver;
-        this.sessionId = driver.getSessionId().toString();
     }
 
-    public String getDebugUrl() {
-        return getSessionDetails(driver).get("browser_url").toString();
+    public String getDebugUrl(JSONObject result) {
+        String buildHash = result.getString("buildHash");
+        String sessionHash = result.getString("sessionHash");
+        return "https://app-automate.browserstack.com/dashboard/v2/builds/" + buildHash + "/sessions/" + sessionHash;
```

Comment:
Wait, this will have problem for people using regional / enterprise dashboard. do we have list of all dashboards that users use ?

Reply:
Resolved on percy web